platformio.ini

```diff
@@ -11,6 +11,7 @@
 [env]
 monitor_speed = 115200 ; must remain at 115200 for improv
 board_build.filesystem = littlefs
+upload_port = /dev/ttyUSB0
```

Comment:
Humm... forgot to remove this.  Possibly unique to my environment.  Thoughts about leaving or removing?